### DISCUSSION TITLE
Wasser/ Luft im Zähler

### DISCUSSION THREAD

**Discussion by surger13** [jomjol/AI-on-the-edge-device#3245] [Q&A]
Hallo zusammen,

ich habe vor ca. 2 Wochen einen zweiten Digitalisierer für meine Wasseruhr gebaut. Die wurde zu dem Zeitpunkt durch den Versorger turnusmäßig gewechselt. Das ganze lief bis gestern relativ stabil und ich war erneut -wie schon beim vorher gebauten Stromzähler- total begeistert von diesem Projekt hier. 
Als ich heute morgen wieder mal geprüft habe wie es läuft, bot sich der Anblick wie auf dem Bild. Es ist nun eine größere Luftblase im Schauglas deren Rand auch noch genau über den digits läuft. Eine Erkennung ist so unmöglich und seit gestern werden keine sinnvollen Werte mehr geliefert. 

Hat das schonmal jemand gehabt? Gibt es einen Trick, den Zähler zu entlüften (oder zu entwässern ;) )? 

Gruß Dirk
![20240921_Luftblase_oZN](https://github.com/user-attachments/assets/d050a3f7-ec7a-4252-87b9-04ea86a10a8f)

**Comment by friedpa** [comment]
Diese Luftblase verschwindet nach einiger Zeit wieder, wenn der Zähler mit Wasser durchströmt wird. Was man einstweilen machen kann ist den Zähler leicht zu neigen, dann verschwindet die Luftblase zum Rand... aber wie schon geschrieben verachwindet diese eh beim Wasserverbrauch von selbst.

**Reply by surger13** [reply]
Danke, dann harre ich mal der Dinge. Neigen ist sowieso nicht, das Teil ist ja verplombt.
